net/tcp/tcp_timer.c

```diff
@@ -470,7 +452,7 @@ void tcp_timer(FAR struct net_driver_s *dev, FAR struct tcp_conn_s *conn,
 #endif
                       /* Update for the next probe */
 
-                      conn->keeptime = clock_systime_ticks();
+                      conn->keeptimer = conn->keepintl;
```

Comment:
```suggestion
                      conn->keeptimer = conn->keepintvl;
```
CI complained about this one

Reply:
Done.